### ISSUE TITLE
webapp:az webapp up

### ISSUE THREAD

**Issue by nordquist** [Azure/azure-cli#17415]
## Describe the bug

**Command Name**
`az webapp up`

**Errors:**
```
The webapp 'happyatwork-admin-ui' is a Windows app. The code detected at 'D:\Work\HaW\Code\happyatwork-admin\admin-ui' will default to 'Linux'. Please create a new app to continue this operation.
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az webapp up`

## Environment Summary
```
Windows-10-10.0.19041-SP0
Python 3.6.8
Installer: MSI

azure-cli 2.21.0

```
## Additional Context

```
az --version
azure-cli                         2.21.0

core                              2.21.0
telemetry                          1.0.6

Python location 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe'
Extensions directory 'C:\Users\nordq\.azure\cliextensions'

Python (Windows) 3.6.8 (tags/v3.6.8:3c6b436a57, Dec 23 2018, 23:31:17) [MSC v.1916 32 bit (Intel)]

Legal docs and information: aka.ms/AzureCliLegal
```

Your CLI is up-to-date.

Please let us know how we are doing: https://aka.ms/azureclihats
and let us know if you're interested in trying out our newest features: https://aka.ms/CLIUXstudy

**Comment by Kotasudhakarreddy** [comment]
@nordquist - this is because the discrepancy b/w the OS of `appservice` and the `deployment code ` your are trying to deploy. Here the `appservice` is of `windows` type and the deployment code form `'D:\Work\HaW\Code\happyatwork-admin\admin-ui` is of `Linux`. Please try with same OS types. If you still face the issue please reply back with `--debug` log.

**Comment by Kotasudhakarreddy** [comment]
@nordquist hope you are unblocked now. Please let us know your response on whether we are good to close this issue.

**Comment by panchagnula** [comment]
closing due to lack of response. Please file a new issue if this is still a problem.
